apache/superset
# style: Sticky header in db modal

### SUMMARY
Made the header in the database modal sticky, so that only the middle content scrolls

### ANIMATED GIF
![stickyHeader](https://user-images.githubusercontent.com/55605634/122299005-0d2e1300-cec3-11eb-8d0a-c00a5938c3a6.gif)

### TESTING INSTRUCTIONS
From the Superset home page:
- Hover over "Data" in the upper navigation bar
- Click "Database"
- Click the "+ Database" button below the navigation bar and to the right
- Select the `SQLite` database
- Click the "Advanced" tab
- Select "Performance" (or any tab that will make the modal scrollable
- Observe sticky header

### ADDITIONAL INFORMATION
